### PR TITLE
Extensions present

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-tls-subcerts.md
Comment (paulwouters) [inline draft-ietf-tls-subcerts.md:470]: while this is technically a change that the WG should approve, as it is fairly obvious the additional restrictions were implied before, I approve of this change.

**Review by paulwouters — APPROVED** [review]

**Review by bifurcation — APPROVED** [review]

**Pull request merged**
